composer.json

```diff
@@ -44,9 +44,13 @@
         "drupal/view_unpublished": "1.x-dev",
         "drush/drush": "^9.2",
         "oomphinc/composer-installers-extender": "^1.1",
+        "symfony/debug": "^3.3.3",
```

Comment:
These version constraints probably won't be an issue for new projects if `drupal/console` wasn't a dev dependency as the version requirements would be gathered in the `create-project` step. You'd normally run into these dependency issues if you were adding `drupal/console` to an existing project. IMO, this scaffold should always be treated that new features are for _new_ projects and not existing ones as that's what we've only supported.

Reply:
Yep you're right. Clearing out the lock file and simply running install with console listed determined the proper versions. Thanks!